phpunit.xml

```diff
@@ -2,14 +2,7 @@
 
 <phpunit
```

Comment:
While we're here changing this file, we could add the [schema](https://schema.phpunit.de/7.5/phpunit.xsd) for this file so we can get IDE completion and warnings.

Reply:
Done!